Userland/Libraries/LibGUI/TextDocument.cpp

```diff
@@ -920,6 +920,70 @@ String ReplaceAllTextCommand::action_text() const
     return m_action_text;
 }
 
+IndentSelection::IndentSelection(TextDocument& document, TextRange const& range)
+    : TextDocumentUndoCommand(document)
+    , m_range(range)
```

Comment:
If you pass the tab-width in as a constructor parameter, you won't have to lazy-initialize it later. Same for unindenting.

Reply:
It is passed in as a constructor parameter now. Let me know if the change I made is what you had in mind.